action.yaml

```diff
@@ -20,6 +20,8 @@ inputs:
     description: "Buildkit version to use with Copa"
   copa-version:
     description: "Copa version to use"
+  output-file:
```

Comment:
let's add `format` here, we can default to `openvex` for now so we can extend this later